### PR TITLE
Gloves of Hugging

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

gloves of north star but with help intent
if you try to not hit something while wearing it it will switch your intent back to help
spawns as arcade prize
r8FNMzW

#### Changelog

:cl:  
rscadd: Added a new arcade prize, the Gloves of Hugging
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
Useless at best  spammy at worst

**Comment by overlord12345** [comment]
good pr

**Comment by ynot01** [comment]
> Useless at best spammy at worst

its called a toy

**Comment by LoliconSlayer** [comment]
Plenty of fucking toys that exist only to be annoyingly spammy with, I see no issue

**Comment by overlord12345** [comment]
whats the sprite like though

**Comment by ynot01** [comment]
> whats the sprite like though

like other toys, identical to its antag counterpart

**Comment by overlord12345** [comment]
would be epic if they had a little heart on them

**Comment by ynot01** [comment]
sprite changes have to be run through ident and anything that i create wil not

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -78,3 +78,17 @@
 	input = replacetext(input, "*", "")
 	if(input)
 		warcry = input
+
+/obj/item/clothing/gloves/rapid/hug
+	name = "Gloves of Hugging"
+	desc = "Just looking at these fills you with an urge to hug the shit out of people."
```
Comment (Hopekz) [inline code/modules/clothing/gloves/miscellaneous.dm:84]:
```suggestion
	desc = "Just looking at these fills you with an urge to hug people!"
```

Comment (monster860) [reply code/modules/clothing/gloves/miscellaneous.dm:84]: this is a christian minecraft server. no swearing on my christian minecraft server.

**Pull request merged**